nilearn/plotting/js_plotting_utils.py

```diff
@@ -5,12 +5,15 @@
 
 import os
 import base64
-import cgi
 import webbrowser
 import tempfile
 import warnings
 import subprocess
 import weakref
+try:
```

Comment:
Maybe a comment to say why this try except? It will make it easier to clean this in the future.

Reply:
Done.